pandas/tests/io/formats/test_style.py

```diff
@@ -376,6 +376,23 @@ def color_negative_red(val):
 
         (df.style.applymap(color_negative_red, subset=idx[:, idx["b", "d"]]).render())
 
+    def test_applymap_subset_multiindex_code(self):
+        codes = np.array([[0, 0, 1, 1], [0, 1, 0, 1]])
```

Comment:
can you add the issue number as a comment and a 1-liner explain

Reply:
This is now done